spotless.importorder

```diff
@@ -0,0 +1,2 @@
+0=java
```

Comment:
While we are at rewriting, please change the formatting from tabs to using 4 spaces, they are much more comfortable to work with and IntelliJ would stop complaining about every file not being formatted correctly:
![image](https://github.com/user-attachments/assets/4f365376-d182-4b72-b852-9213f76c5ea0)

Reply:
As far as I know, @dfuchss prefers tabs

Reply:
For me spaces are fine too. Just configure it in the formatter :)